packages/common/src/services/integrations/AccountService.ts

```diff
@@ -31,18 +31,21 @@ export type AccountServiceFeatures = {
   readonly canUpdatePaymentMethod: boolean;
   readonly canShowReceipts: boolean;
   readonly hasSocialURLs: boolean;
-  readonly hasProfiles: boolean;
```

Comment:
Do you think we can remove it? We don't have profiles for Cleeng.

Reply:
`ProfileService` is named and only registered for the JWP integration. So when using Cleeng, the service isn't registered. The ProfileController knows this and exposes an `isEnabled` method to prevent unnecessary calls and waiting for disabled queries.